docs/compression_algorithms/CompressWeights.md

```diff
@@ -61,10 +61,11 @@ nncf_dataset = nncf.Dataset(data_source, transform_fn)
 compressed_model = compress_weights(model, mode=CompressWeightsMode.INT4_SYM, ratio=0.8, dataset=nncf_dataset) # model is openvino.Model object
 ```
 
-- Accuracy of the 4-bit compressed models also can be improved by using AWQ algorithm over data-based mixed-precision algorithm. It is capable to equalize some subset of weights to minimize difference between
+- Accuracy of the 4-bit compressed models also can be improved by using AWQ algorithm or Scale Estimation algorithm over data-based mixed-precision algorithm. It is capable to equalize some subset of weights to minimize difference between
 original precision and 4-bit.
-Below is the example how to compress 80% of layers to 4-bit integer with a default data-based mixed precision algorithm and AWQ.
-It requires to set `awq` to `True` additionally to data-based mixed-precision algorithm.
+Below is the example how to compress 80% of layers to 4-bit integer with a default data-based mixed precision algorithm and AWQ with Scale Estimation.
+It requires to set `awq` to `True` and `scale_estimation` to `True` additionally to data-based mixed-precision algorithm.
+Both algorithms (AWQ and Scale Estimation) works independently.
```

Comment:
"Both algorithms (AWQ and Scale Estimation) works independently." - I think it's unclear and I would rephrase it. How about "Both algorithms, AWQ and Scale Estimation, can be enabled together or separately."?

Reply:
Done

Reply:
> Both algorithms

remove unnecessary dot at the end of 'Both algorithms, AWQ and Scale Estimation, can be enabled together or separately..'

Reply:
Done